src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import {
+  openSelectAppointmentPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { focusElement } from 'platform/utilities/ui';
+import FormButtons from '../components/FormButtons';
+import { getDateTimeSelect } from '../utils/selectors';
+import DateTimeSelectField from '../components/DateTimeSelectField';
+
+const pageKey = 'selectDateTime';
```

Comment:
It hurts me that we don't have these page keys stored in a central location so they can also be used in the flow-control files, but Jeff already mentioned he knows theres a lot of throwing reused strings around this app right now.

Reply:
Agreed.  We should have a const file for these.